mocca-client/src/main/java/com/paypal/mocca/client/MoccaFeignEncoder.java

```diff
@@ -99,9 +99,7 @@ void validateVariables(Object[] parameters, RequestTemplate template) {
         Set<ConstraintViolation<Object>> violationSet = validator.forExecutables().validateParameters(client, method, parameters);
 
         if (violationSet.size() > 0) {
-            throw new EncodeException("Constraint violations found in one or more request parameters",
-                    new ConstraintViolationException(violationSet)
-            );
+            throw new ConstraintViolationException(violationSet);
```

Comment:
Is it clear that a constraint violation has occurred without the message? Just wondering.

Reply:
There will still be a message. There is always the default message, which has to the field that failed validation and the validation failure. There is also an opportunity for the application owner to customize the messages when they add Bean Validations annotations.